### ISSUE TITLE
Select: required field has text communicating field is required and label associated to labeled control (3.3.2, )

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#864]
### Expected behavior

Input field (or its label) has text or attribute indicating it is required and is programmatically connected to the input field. The label is associated to its labeled control. In this case the button. 
### Actual behavior

Red star inserted for text element preceding single select button. Someone listening hears "single select mode star" and nothing to indicate the field is required. 

The label is not associated  to the input field (i.e., it doesn't have the id for the "for" attribute on the label).
